### PR TITLE
Register `xtensor-julia`

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
It seems that CI requires the git repository URL to end with `.jl` which is weird since it is not a requirement for `Pkg.clone`.
Also it does not seem to be specified in the documentation that registered git repos should follow this pattern.

**Comment by SylvainCorlay** [comment]
@tkelman is there any blocker to get this in?

**Comment by SylvainCorlay** [comment]
:sparkles:

**Comment by tkelman** [comment]
whoa, new package registrations should remain open for 24 hours

**Comment by KristofferC** [comment]
Where is this documented?

**Comment by tkelman** [comment]
It's now included in the attobot text for new package registrations that use attobot, but it's been the policy applied for initial feedback in multiple past discussions. I'd add it to the readme if that were visible.

File: Xtensor/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+BinDeps 0.4.2
+Compat 0.21
```
Comment (tkelman) [inline Xtensor/versions/0.0.1/requires]: you don't appear to be actually using Compat anywhere

Comment (SylvainCorlay) [inline Xtensor/versions/0.0.1/requires]: Thanks, I did at some point and did not remove the line.

**Pull request merged**